client/components/RoomList.jsx

```diff
@@ -14,31 +23,67 @@ const RoomList = React.createClass({
 
   componentDidMount() {
     $('.ui.accordion').accordion();
+    Meteor.call("get.room.counts", (error, response) => {
+      this.setState(response);
```

Comment:
changed the name to something that is much easier understood, The "get.room.counts" method simply retrieved the number of unread messages for that room and user, Now renamed as 'get.room.unreadMsgCount"